pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +273,18 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) isDockerAPIVersionCompatible(input string) bool {
```

Comment:
Ok, that's fine. We should probably either get rid of `nativeExecSupportExists` or update it to use this func (and then create `parallelPullSupportExists`)